tests/413-verifiable-presentations.js

```diff
@@ -107,24 +121,31 @@ describe('VP - Enveloped Verifiable Credentials', function() {
         'Failed to reject a VP containing an enveloped VC with a missing ' +
         '`type`.');
         this.skip();
+        // TODO implement Enveloped Tagging
       });
 
       it('The id value of the object MUST be a data: URL [RFC2397] that ' +
         'expresses a secured verifiable credential using an enveloping ' +
         'security scheme, such as Securing Verifiable Credentials using JOSE ' +
         'and COSE [VC-JOSE-COSE].', async function() {
         this.test.link = `https://w3c.github.io/vc-data-model/#enveloped-verifiable-credentials:~:text=The%20id%20value%20of%20the%20object%20MUST%20be%20a%20data%3A%20URL%20%5BRFC2397%5D%20that%20expresses%20a%20secured%20verifiable%20credential%20using%20an%20enveloping%20security%20scheme%2C%20such%20as%20Securing%20Verifiable%20Credentials%20using%20JOSE%20and%20COSE%20%5BVC%2DJOSE%2DCOSE%5D.`;
-        // TODO: implement test
-        this.test.cell.skipMessage = 'TBD';
+        issuedVc.should.have.property('id').that.does
+          .include('data:application/vc+jwt',
+            `Expecting id field to be a data: URL [RFC2397].`);
+        const extractedCredential = extractIfEnveloped(issuedVc);
+        shouldBeCredential(extractedCredential);
         this.skip();
```

Comment:
why do we assert here and then skip?

Reply:
@PatStLouis guessing the above `this.skip()` just got overlooked?

Reply:
The problem here is because this test will also be executed on a non enveloped credential. I put the skip there until we have some way to tag implementations, I should of left a comment!